ash/src/device.rs

```diff
@@ -64,6 +68,274 @@ impl Device {
     }
 }
 
+/// Vulkan core 1.4
+impl Device {
+    #[inline]
+    pub fn fp_v1_4(&self) -> &crate::DeviceFnV1_4 {
+        &self.device_fn_1_4
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdSetLineStipple.html>
+    #[inline]
+    #[doc(alias = "vkCmdSetLineStipple")]
+    pub unsafe fn cmd_set_line_stipple(
+        &self,
+        command_buffer: vk::CommandBuffer,
+        line_stipple_factor: u32,
+        line_stipple_pattern: u16,
+    ) {
+        (self.device_fn_1_4.cmd_set_line_stipple)(
+            command_buffer,
+            line_stipple_factor,
+            line_stipple_pattern,
+        )
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkMapMemory2.html>
+    #[inline]
+    #[doc(alias = "vkMapMemory2")]
+    pub unsafe fn map_memory2(
+        &self,
+        memory_map_info: &vk::MemoryMapInfo<'_>,
+    ) -> VkResult<*mut ffi::c_void> {
+        let mut data = mem::MaybeUninit::uninit();
+        (self.device_fn_1_4.map_memory2)(self.handle, memory_map_info, data.as_mut_ptr())
+            .assume_init_on_success(data)
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkUnmapMemory2.html>
+    #[inline]
+    #[doc(alias = "vkUnmapMemory2")]
+    pub unsafe fn unmap_memory2(
+        &self,
+        memory_unmap_info: &vk::MemoryUnmapInfo<'_>,
+    ) -> VkResult<()> {
+        (self.device_fn_1_4.unmap_memory2)(self.handle, memory_unmap_info).result()
+    }
```

Comment:
Of all the stabilized extensions, only `VK_KHR_map_memory2` was missing a wrapper implementation in Ash. I'm considering to retroactively add it.